packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -159,7 +160,18 @@ export class SummarizerNode implements IRootSummarizerNode {
 			}
 		}
 
-		const result = await this.summarizeInternalFn(fullTree, true, telemetryContext);
+		const incrementalContext: IIncrementalContext = {
+			sequenceNumber: this.wipReferenceSequenceNumber ?? this._changeSequenceNumber,
+			previousSequenceNumber: this._latestSummary?.referenceSequenceNumber ?? 0,
```

Comment:
What's `previousSequenceNumber`?

Reply:
The rename should be previousSuccessfulSummaryNumber

Reply:
Maybe `lastAckedSummarySequenceNumber`